parser/src/stateful/decode.rs

```diff
@@ -726,7 +726,7 @@ where
 
         if header.tag == Tag(0x0028, 0x0103) {
             //Pixel Representation is not 0, so 2s complement (signed)
-            self.signed_pixeldata = Some(vec[0] != 0);
+            self.signed_pixeldata = vec.get(0).map(|rep| *rep != 0);
```

Comment:
Addressing warning from Clippy. 
```suggestion
            self.signed_pixeldata = vec.first().map(|rep| *rep != 0);
```